src/adv.py

```diff
@@ -1,29 +1,33 @@
 from room import Room
+from player import Player
 
 # Declare all the rooms
 
 room = {
     'outside':  Room("Outside Cave Entrance",
-                     "North of you, the cave mount beckons"),
+                     "North of you, the cave mount beckons",None, None, None, None),
```

Comment:
Instead of adding 4 'None's to the end of each room we create, we can set these parameters to be 'None' by default if we don't pass in any values (we would do this in the room.py where we create the room class)